testsuite/forge-cli/src/README.md

```diff
@@ -7,37 +7,70 @@ custom_edit_url: https://github.com/aptos-labs/aptos-core/edit/main/testsuite/fo
+example, to deploy a local validator swarm, run:
 
 ```
-cargo run -p forge-cli -- --suite "run_forever" --num-validators 3 test local-swarm
+cargo run -p forge-cli -- --suite "run_forever" --num-validators 4 test local-swarm
 ```
 
-This will start a local swarm of 3 validators, each running in their own process. The
-swarm will run forever, unless manually killed. The output will display the locations
-of the swarm files (e.g., the genesis files, logs, node configurations, etc.) and the
-commands that were run to start each node. The process id (PID) of each node is also
-displayed when it starts.
+This will start a local network of 4 validators, each running in their own process. The
+network will run forever, unless manually killed. The output will display the locations
+of the validator files (e.g., the genesis files, logs, node configurations, etc.) and the
+commands that were run to start each node. The process id (PID) of each node and
+server addresses (e.g., REST APIs) are also displayed when it starts. For example, if you
+run the above command you should see:
 
-Using the information from the above command, you could stop a single node and restart
-it, e.g., run:
+```
+...
+2022-09-01T15:41:27.228289Z [main] INFO crates/aptos-genesis/src/builder.rs:462 Building genesis with 4 validators. Directory of output: "/private/var/folders/dx/c0l2rrkn0656gfx6v5_dy_p80000gn/T/.tmpq9uPMJ"
+...
+2022-09-01T15:41:28.090606Z [main] INFO testsuite/forge/src/backend/local/swarm.rs:207 The root (or mint) key for the swarm is: 0x4A7ED...
+...
+2022-09-01T15:41:28.094800Z [main] INFO testsuite/forge/src/backend/local/node.rs:129 Started node "0" (PID: 78939) with command: ".../aptos-core/target/debug/aptos-node" "-f" "/private/var/folders/dx/c0l2rrkn0656gfx6v5_dy_p80000gn/T/.tmpq9uPMJ/0/node.yaml"
+2022-09-01T15:41:28.094825Z [main] INFO testsuite/forge/src/backend/local/node.rs:137 Node "0" REST API is listening at: 127.0.0.1:50271/v1
+2022-09-01T15:41:28.094838Z [main] INFO testsuite/forge/src/backend/local/node.rs:142 Node "0" Inspection service is listening at 127.0.0.1:50273
+...
+```
+
+Using the information from this output, you could stop a single node and restart
+it, e.g., stop and restart node `0`:
 
 ```
-kill -9 <Node PID>
-cargo run -p aptos-node -- -f <Location to the node configuration file outputted above>
+kill -9 78939
+cargo run -p aptos-node -- -f "/private/var/folders/dx/c0l2rrkn0656gfx6v5_dy_p80000gn/T/.tmpq9uPMJ/0/node.yaml"
 ```
 
-To see all tool usage options, run:
+## Faucet and Minting
+
+In addition, the swarm output also displays the root (or mint) key for the network. This allows
+you to run a local faucet and start minting test tokens in the network. For this, simply run the
+faucet command using the mint key and point it to the REST API of one of the nodes, e.g. node `0`:
+
 ```
-cargo run -p forge-cli --help
+cargo run --bin aptos-faucet -- -c TESTING --mint-key 0x4A7ED... -s http://127.0.0.1:50271 -p 8081   
```

Comment:
Similarly, I'd refer to the <REST API URL> above, especially having to call out http://

Should we print that out in the original log?

Reply:
Updated the example and the print to include http://